helm/chaos-mesh/templates/chaos-daemon-rbac.yaml

```diff
@@ -16,7 +16,7 @@ kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: {{ .Release.Name }}-chaos-daemon-target-namespace
-  namespace: chaos-testing
+  namespace: {{ .Release.Namespace }}
```

Comment:
Would it be better to change to "namespace: {{ .Release.Namespace | quote }}"  ?

Reply:
Sure, done, thanks!